I fixed the CI error in commits up to 6d5044e.
https://github.com/ktyubeshi/rspolib/actions/runs/20712823160

There was an issue where, on Python 3.8, the pip version check would report an error:

##[error][notice] A new release of pip is available: 21.1.1 -> 25.0.1

I tried options such as pinning the pip version (recently added to actions/setup-python), but as described in the issue below, the runner image no longer supports Python 3.8. Therefore, I suppressed the error by disabling the version check.
https://github.com/actions/setup-python/issues/862

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -161,7 +161,7 @@ jobs:
       - name: Publish (dry run)
         run: |
           cd rust
-          cargo package -v --dry-run
+          cargo publish -v --dry-run
 
   release-rust-crate:
     name: Release Rust crate

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -20,7 +20,6 @@ jobs:
         uses: actions/setup-python@v6
         with:
           python-version: "3.12.1"
-          pip-version: "25.0.1"
       - name: Install dependencies
         run: |
           pip install --upgrade pip
@@ -82,7 +81,6 @@ jobs:
           PIP_DISABLE_PIP_VERSION_CHECK: 1
         with:
           python-version: ${{ matrix.python-version }}
-          pip-version: "25.0.1"
       - uses: actions/cache@v4
         with:
           path: ~/.cache/pip
@@ -160,10 +158,10 @@ jobs:
         uses: hecrj/setup-rust-action@v2
         with:
           rust-version: stable
-      - name: Package
+      - name: Publish (dry run)
         run: |
           cd rust
-          cargo package -v
+          cargo package -v --dry-run
 
   release-rust-crate:
     name: Release Rust crate

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+## 2026-01-08 - [0.1.2]
+
+### Bug fixes
+
+- Incomplete entries (msgctxt/msgid without msgstr) could trigger a panic when a new entry begins.
+- Previous-translation continuation lines (#| "...") could be accepted without a preceding #| msgid/msgctxt/msgid_plural context.
+
 ## 2025-01-20 - [0.1.1]
 
 - Add MSRV.
@@ -14,6 +21,7 @@ First beta release.
 
 Alpha releases.
 
+[0.1.2]: https://github.com/mondeja/rspolib/compare/v0.1.1...v0.1.2
 [0.1.1]: https://github.com/mondeja/rspolib/compare/v0.1.0...v0.1.1
 [0.1.0]: https://github.com/mondeja/rspolib/compare/v0.0.5...v0.1.0
 [0.0.5]: https://github.com/mondeja/rspolib/compare/v0.0.1...v0.0.5

--- a/python/Cargo.toml
+++ b/python/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "py-rspolib"
-version = "0.1.1"
+version = "0.1.2"
 edition = "2021"
 readme = "README.md"
 description = "Python bindings for the Rust crate rspolib."

--- a/rust/Cargo.toml
+++ b/rust/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "rspolib"
-version = "0.1.1"
+version = "0.1.2"
 rust-version = "1.66.1"
 edition = "2021"
 readme = "../README.md"

